content/docs/Tutorials/goFSH/_index.md

```diff
@@ -41,7 +41,7 @@ The current pre-release of GoFSH does not always produce the most efficient FSH
 
 ### Step 5: Run SUSHI (Optional)
 
-Now that you have generated FSH definitions, you can run SUSHI on those definitions to recreate your original input to GoFSH. First, ensure you have [SUSHI installed](/docs/sushi/installation), then download the **config.yaml** file given below:
+Now that you have generated FSH definitions, you can run SUSHI on those definitions to recreate your original input to GoFSH. First, ensure you have [SUSHI installed](/docs/sushi/installation), then download the **sushi-config.yaml** file given below:
 {{% show-file src="config" download="bottom" %}}
```

Comment:
This file isn't rendering for me, but I think the source file name just needs to be updated at the top of the file to reflect the file name change to `sushi-config.yaml`.